navigation.yaml

```diff
@@ -1509,3 +1509,20 @@ cube:
         - title: Thank you
           path: /cube/thank-you
           hidden: True
+
+gcp:
+  title: GCP
+  path: /gcp
+
+  children:
+    - title: Pro
```

Comment:
You are missing an overview I believe as so there is no second-level navigation.

Reply:
Fixed